lib/src/react_client/event_helpers.dart

```diff
@@ -0,0 +1,701 @@
+/// but no fields need to be set.
+SyntheticMouseEvent getEmptyMouseEvent() => createSyntheticMouseEvent();
+
+/// Returns an empty [SyntheticPointerEvent] with only the defaults set.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticPointerEvent getEmptyPointerEvent() => createSyntheticPointerEvent();
+
+/// Returns an empty [SyntheticTouchEvent] with only the defaults set.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticTouchEvent getEmptyTouchEvent() => createSyntheticTouchEvent();
+
+/// Returns an empty [SyntheticTransitionEvent] with only the defaults set.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticTransitionEvent getEmptyTransitionEvent() => createSyntheticTransitionEvent();
+
+/// Returns an empty [SyntheticAnimationEvent] with only the defaults set.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticAnimationEvent getEmptyAnimationEvent() => createSyntheticAnimationEvent();
+
+/// Returns an empty [SyntheticUIEvent] with only the defaults set.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticUIEvent getEmptyUIEvent() => createSyntheticUIEvent();
+
+/// Returns an empty [SyntheticWheelEvent] with only the defaults set.
+///
+/// This utility can be useful when a test just needs access to an event object,
+/// but no fields need to be set.
+SyntheticWheelEvent getEmptyWheelEvent() => createSyntheticWheelEvent();
+
+/// Returns a newly constructed [SyntheticEvent] instance.
+///
+/// In addition to creating empty instances (by invoking without any parameters) or completely custom instances
+/// by using the named parameters, this function will merge previously existing [baseEvent]s with the named parameters provided.
+/// The named parameters takes precedence, and therefore can be used to override specific fields on the [baseEvent].
+SyntheticEvent createSyntheticEvent({
+  SyntheticEvent baseEvent,
+  bool bubbles,
+  bool cancelable,
+  dynamic currentTarget,
+  bool defaultPrevented,
+  dynamic preventDefault,
+  dynamic stopPropagation,
+  num eventPhase,
+  bool isTrusted,
+  dynamic nativeEvent,
+  dynamic target,
+  num timeStamp,
+  String type,
+}) {
+  return SyntheticEvent(
+    bubbles ?? baseEvent?.bubbles ?? true,
+    cancelable ?? baseEvent?.cancelable ?? true,
+    currentTarget ?? baseEvent?.currentTarget,
+    defaultPrevented ?? baseEvent?.defaultPrevented ?? false,
+    preventDefault ?? baseEvent?.preventDefault ?? () {},
+    stopPropagation ?? baseEvent?.stopPropagation ?? () {},
+    eventPhase ?? baseEvent?.eventPhase,
+    isTrusted ?? baseEvent?.isTrusted ?? true,
```

Comment:
I think this should default to `false`

https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted
> The isTrusted read-only property of the Event interface is a Boolean that is true when the event was generated by a user action, and false when the event was created or modified by a script or dispatched via EventTarget.dispatchEvent().

Reply:
Ahh yep, thank you!